phpunit/class-wp-navigation-block-renderer-test.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Test the block WP_Navigation_Block_Renderer class.
+ *
+ * @package Gutenberg
+ */
+
+class WP_Navigation_Block_Renderer_Test extends WP_UnitTestCase {
```

Comment:
@scruffian I think we should move this test into the `phpunit/blocks` directory. It is specific to a specific block and thus should be managed within the Gutenberg repo. 

See https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/back-merging-to-wp-core.md#ignored-directoriesfiles

Reply:
Done in https://github.com/WordPress/gutenberg/pull/58247